doc/overview.rst

```diff
@@ -71,7 +71,7 @@ First, we need to import a few other packages:
     import pygmt         # For plotting maps
     import xarray as xr  # For manipulating grids
 
-Now we can download and open co-located topography and geoid grids using
+Now we can download and open :co-located:`co-located grids` topography and geoid grids using
 :mod:`ensaio` and :mod:`xarray`:
```

Comment:
```suggestion
Now we can download and open :term:`co-located grids` of topography and geoid
using :mod:`ensaio` and :mod:`xarray`:
```

Reply:
The command for linking to the glossary is `:term:`. Using that requires rephrasing a bit to avoid the duplicated "grids".

Reply:
Thank you for explaining! Sometimes I am confused about where I have to substitute text and where is an actual command 😕